### ISSUE TITLE
bug in the UI

### ISSUE THREAD

**Issue by amichaiu** [clun/ff4j#228]
I have an issue with the UI of the expression.
if in the DB you see this (off_for=*&on_for=IT123):
![image](https://cloud.githubusercontent.com/assets/29042383/26551029/42b5b8ac-4489-11e7-9a8f-0e622f771a25.png)
in the UI I see (off_for=*, on_for=IT123): 
![image](https://cloud.githubusercontent.com/assets/29042383/26551121/a55b7bea-4489-11e7-9310-e1f2d433bd16.png)
and if I click on update without updating anything, in the DB it becomes:
![image](https://cloud.githubusercontent.com/assets/29042383/26551179/dea9aee4-4489-11e7-9143-73df48c50d47.png)
and also in the UI it cuts it:
![image](https://cloud.githubusercontent.com/assets/29042383/26551204/fccca322-4489-11e7-96f8-2384ecb6f5b9.png)
looks like that although in the UI it shows "," - for having it work you need to put ";" (off_for=*;on_for=IT123)
is this a known bug? anyway to fix it?
Thanks

**Comment by clun** [comment]
Expression in DB `(off_for=&on_for=IT123):` is tranformed to `(off_for=, on_for=IT123):` in the UI

This "issue" seems to be related to multiple init_param. (it's a map). In the DB is serialized with `&`characters. When displayed in the UI it's separated by ','. Problem occur when parsing the field in the UI it seems.

**Comment by clun** [comment]
Switching to SpringJDBC, SameDb, Same Data

 ```java
ff4j.setFeatureStore(new FeatureStoreSpringJdbc(dbcpDataSource));
ff4j.setPropertiesStore(new PropertyStoreSpringJdbc(dbcpDataSource));
ff4j.setEventRepository(new EventRepositorySpringJdbc(dbcpDataSource));
ff4j.audit(true);
 ```
Again ...cannot reproduce BUG....

Am i missing something there ?

**Comment by clun** [comment]
Any update on this @amichaiu  ?

**Comment by amichaiu** [comment]
Hi.
I will try to do a change on the latest version that fixes the problem and let you know. I did a change on our version (1.3.1) but I think it would be better for the community to have a fixed updated version.
Mainly the fix I did in our version is to show the parameters of the map with “;” between them (like when you pars it) rather than the default “,” that the map uses.
I will do the change on the new version later, see it works and try to create a pull request (or send you the 1 or 2 files with the change)
Thanks
Amichai
